cmd/controlplane/webhooks.go

```diff
@@ -35,10 +35,11 @@ func newWebhooksServerCommand() *cobra.Command {
 			ctx := cmd.Context()
 
 			version := versionpkg.GetVersion()
-			log.WithFields(log.Fields{
-				"version": version.Version,
-				"commit":  version.GitCommit,
-			}).Info("Starting Kargo Webhooks Server")
+			log := logging.LoggerFromContext(ctx)
```

Comment:
See [this comment](https://github.com/akuity/kargo/pull/1791/files#r1556679011) -- same concern.

Reply:
I'll handle these after a rebase, looks like stuff has moved around a bit.